src/main/java/it/pagopa/pn/client/b2b/pa/polling/impl/PnPollingServiceValidationStatusAccepted.java

```diff
@@ -0,0 +1,90 @@
+package it.pagopa.pn.client.b2b.pa.polling.impl;
+
+
+import it.pagopa.pn.client.b2b.pa.config.PnB2bClientTimingConfigs;
+import it.pagopa.pn.client.b2b.pa.generated.openapi.clients.externalb2bpa.model.NewNotificationRequestStatusResponseV23;
+import it.pagopa.pn.client.b2b.pa.polling.design.PnPollingStrategy;
+import it.pagopa.pn.client.b2b.pa.polling.design.PnPollingTemplate;
+import it.pagopa.pn.client.b2b.pa.polling.dto.PnPollingResponseV23;
+import it.pagopa.pn.client.b2b.pa.service.IPnPaB2bClient;
+import it.pagopa.pn.client.b2b.pa.service.impl.PnPaB2bExternalClientImpl;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.Callable;
+import java.util.function.Predicate;
+@Service(PnPollingStrategy.VALIDATION_STATUS_ACCEPTATION)
```

Comment:
lasciare solo VALIDATION_STATUS
gestire come discusso assieme in call con i valori in timingForTimeline: 
 ACCEPTED_VALIDATION(10,0/*TEMPO CORRETT*/),
 REFUSED_VALIDATION(10,0/*TEMPO CORRETT*/),
_VALIDATION sarà aggiunto dal componente in fase di chiamata alla classe per recuperare i valori (l'utilizzatore del polling chiamerà solo con ACCEPTED e REFUSED)

Reply:
abbiamo apportato le modifiche concordate